### PR TITLE
Create your own Enumerable

### PR DESCRIPTION
**Pull request by Alexr16**
### Project requirements

- Created a class `MyList` that has an instance variable `@list`.
- In `MyList` implemented a method `#each` that yields successive members of `@list` and uses the `MyEnumerable` module.
- Created a module `MyEnumerable` that implements the following methods:
  - `#all?`
  - `#any?`
  - `#filter`
- Each class and module has a separate .rb file.
- Wrote comments to test solution.

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Test _any method
+# puts my_list._any { |e| e == 2 }
+# p my_list._any { |e| e == 5 }
+
+# Test _all method
+# p my_list._all { |e| e < 5 }
+# p my_list._all { |e| e > 5 }
+
+# Test _filter method
+# p my_list._filter { |e| e.even? }
```
Comment (OLIPLICHE) [inline my_list.rb]: - [ ] Please clean this comment of code

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Great Job

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb (hunk 1 above)
Comment (Alexr16) [reply my_list.rb]: @OLIPLICHE Thank you for your feedback.
The requirement was done.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Dangelo,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Implemented required enumerable modules
- Ruby syntax correctly used

<br>
Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**